dotCMS/src/main/java/com/dotcms/storage/JsonWriterDelegate.java

```diff
@@ -0,0 +1,29 @@
+package com.dotcms.storage;
+
+import com.dotmarketing.util.Logger;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+/**
+ * Simple implementation that basically converts the object into a json
+ * @author jsanca
+ */
+public class JsonWriterDelegate implements ObjectWriterDelegate {
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
+
+    public JsonWriterDelegate() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unnecessary constructors - the compiler will generate these for you](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5547873)